<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="English" country="Belgium"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mainNewFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_New.svg</normaloff>:/tb/assets/mainTB_New.svg</iconset>
   </property>
   <property name="text">
    <string>New file</string>
   </property>
   <property name="toolTip">
    <string>Create new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainOpenFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Open.svg</normaloff>:/tb/assets/mainTB_Open.svg</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="toolTip">
    <string>Open existing file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainSaveFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Save.svg</normaloff>:/tb/assets/mainTB_Save.svg</iconset>
   </property>
   <property name="text">
    <string>Save file</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainSaveFileAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_SaveAs.svg</normaloff>:/tb/assets/mainTB_SaveAs.svg</iconset>
   </property>
   <property name="text">
    <string>File save as</string>
   </property>
   <property name="toolTip">
    <string>Save file under a different name or in a different location</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Copy.svg</normaloff>:/tb/assets/mainTB_Copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy selected text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Cut.svg</normaloff>:/tb/assets/mainTB_Cut.svg</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut selected text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Paste.svg</normaloff>:/tb/assets/mainTB_Paste.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste from clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditSelectAll">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_SelectAll.svg</normaloff>:/tb/assets/mainTB_SelectAll.svg</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="toolTip">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Undo.svg</normaloff>:/tb/assets/mainTB_Undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo last action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Redo.svg</normaloff>:/tb/assets/mainTB_Redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo last action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainAppQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Quit_2.svg</normaloff>:/tb/assets/mainTB_Quit_2.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="mainAppInfo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Info.svg</normaloff>:/tb/assets/mainTB_Info.svg</iconset>
   </property>
   <property name="text">
    <string>_dummy_</string>
   </property>
   <property name="toolTip">
    <string>_dummy_</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainOptionsSync">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_Sync.svg</normaloff>:/tb/assets/mainTB_Sync.svg</iconset>
   </property>
   <property name="text">
    <string>Sync. scrolling</string>
   </property>
   <property name="toolTip">
    <string>Sync. scrolling of editor and viewer</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditFormatBold">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_FormatBold.svg</normaloff>:/tb/assets/mainTB_FormatBold.svg</iconset>
   </property>
   <property name="text">
    <string>Format Bold</string>
   </property>
   <property name="toolTip">
    <string>Format selection Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditFormatItalic">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_FormatItalic.svg</normaloff>:/tb/assets/mainTB_FormatItalic.svg</iconset>
   </property>
   <property name="text">
    <string>Format Italic</string>
   </property>
   <property name="toolTip">
    <string>Format selection Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditFormatCode">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_FormatCode.svg</normaloff>:/tb/assets/mainTB_FormatCode.svg</iconset>
   </property>
   <property name="text">
    <string>Format as Code</string>
   </property>
   <property name="toolTip">
    <string>Format selection as Code</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditFormatHighlight">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_FormatHighlight.svg</normaloff>:/tb/assets/mainTB_FormatHighlight.svg</iconset>
   </property>
   <property name="text">
    <string>Highlight</string>
   </property>
   <property name="toolTip">
    <string>Highlight selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mainEditFormatStrikethrough">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tb/assets/mainTB_FormatStrikethrough.svg</normaloff>:/tb/assets/mainTB_FormatStrikethrough.svg</iconset>
   </property>
   <property name="text">
    <string>Format Strikethrough</string>
   </property>
   <property name="toolTip">
    <string>Format selection Strikethrough</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
